tests/Functional/BaseFunctionalTest.php

```diff
@@ -84,7 +84,17 @@ protected function getCommand(
         $queueClient->expects(self::once())->method('getJob')->willReturn($job);
         $queueClient->expects(self::any())->method('getJobFactory')->willReturn($jobFactory);
         $queueClient->expects(self::any())->method('updateJob')->willReturn([]);
-        $queueClient->expects(self::any())->method('patchJob')->willReturn([]);
+        $queueClient->expects(self::any())->method('patchJob')->willReturn(
```

Comment:
Ta `getCommand` metoda je dobry peklo - sdilena factory vselijakych mocku pro hromadu testu. Je to takle "bezpecny"? V podstate ve vesech tech testech ted zacne `patchJob` vracet tenhle result.